samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -204,6 +204,18 @@
                 "description": "A value indicating whether personally identifiable information (PII) redaction is requested. Will only be performed if a Text Analytics Key and Region is provided."
             }
         },
+        "ConversationPiiRedaction": {
+            "defaultValue": "None",
+            "type": "String",
+            "allowedValues": [
+                "None",
+                "DoNotIncludeAudioRedaction",
+                "IncludeAudioRedaction"
```

Comment:
IncludeAudioRedaction does not redact audio yet, or?
I'd hide it to avoid confusion

Reply:
right

Reply:
For now just having an enable only. If we expose the TA outputs directly, we can enable it by default.

Reply:
This is enabled now.